### ISSUE TITLE
Multiple token for same provider cannot have different name and logo, so it’s not reconizable

### ISSUE THREAD

**Issue by x2005254** [freeotp/freeotp-ios#339]
I have to add multiple token for same provider but different account, so i need to have multiple otp, identified by an easy nickname and logo, but I soon I scan a new qr code from the same provider and i put a different logo, otp is changing all my logo for the same provider, it could be great if we can rename also the display name off all otp token we have after adding or during

**Comment by Open-Source-IT-4-Development** [comment]
A renaming option would be great indeed. And/or another possibilty to add a note to an entry. The need for renaming is so obvious that I am kind of nervous regarding maintanance of the (great!) project - and tjis nervousness leads to the mich debated idea of export.

**Comment by cron2** [comment]
yes, please.  I just started migrating access from FreeOTP/Android to FreeOTP/IOS and not being able to properly label tokens on the IOS side is a major shortcoming.
